docs/install/install.sh.md

```diff
@@ -4,20 +4,27 @@ and macOS.
 
 To install, run:
 
-```bash
+```console
```

Comment:
Hi @aslilac we prefer using `shell` for the commands we expect users to copy and paste onto their terminals and `console` when we are also showing the output of a command. So I suggest to change accordingly and use `shell` on these code blocks.